iron-list.html

```diff
@@ -1635,14 +1656,13 @@
       }
 
       var physicalItem = this._physicalItems[this._getPhysicalIndex(idx)];
-      var SECRET = ~(Math.random() * 100);
       var model = physicalItem._templateInstance;
       var focusable;
 
       // set a secret tab index
-      model.tabIndex = SECRET;
+      model.tabIndex = SECRET_TABINDEX;

```

Comment:
In https://github.com/PolymerElements/iron-list/blob/ba4d633a7ad83f3fceb0a89fdaf2aa95aab8880f/iron-list.html#L1670, need to update to `SECRET_TABINDEX`